actions/platform/chrome.py

```diff
@@ -0,0 +1,100 @@
+import json
+
+from actions.slot_match import FuzzySlotMatchOption
+from common.requests import send_console_request
+
+
+async def get_user(tracker):
+    # struggles to be a json response, manually doing it here
+    resp = await send_console_request(
+        "chrome-service",
+        "/api/chrome-service/v1/user",
+        tracker,
+        "get",
+    )
+
+    response, content = resp
+
+    return response, json.loads(content)
+
+
+async def create_service_options(tracker) -> list[FuzzySlotMatchOption]:
+    response, content = await get_generated_services(tracker)
+    if response.ok:
+        services = parse_generated_services(content)
+
+        options = []
+        for service in services:
+            if "href" not in service:
+                # path needs to be here to be favorited
+                continue
+            options.append(convert_service_to_option(service))
+
+        return options
+    else:
+        # failed to reach the chrome service
+        return []
+
+
+def parse_generated_services(content):
+    services = []
+    for category in content:
+        for link in category["links"]:
+            group_title = link.get("title")
+            if "isGroup" in link and link["isGroup"]:
+                for sublink in link["links"]:
+                    if "isExternal" in sublink and sublink["isExternal"]:
+                        # not really a service
+                        continue
+                    service = {"group": group_title}
+
+                    if "href" in sublink:
+                        service["href"] = sublink["href"]
+                    if "title" in sublink:
+                        service["title"] = sublink["title"]
+                    if "appId" in sublink:
+                        service["app_id"] = sublink["appId"]
+                    service["alt_title"] = sublink.get("alt_title", [])
+                    services.append(service)
+    return services
+
+
+async def get_generated_services(tracker):
+    return await send_console_request(
+        "chrome-service",
+        "/api/chrome-service/v1/static/stable/prod/services/services-generated.json",
+        tracker,
+        "get",
+    )
+
+
+def convert_service_to_option(service):
+    value = {"group": service["group"]}
+    synonyms = [service["href"]]
+    value["href"] = service["href"]
+
+    if "title" in service:
+        value["title"] = service["title"]
+        synonyms.append(service["title"])
+    if "appId" in service:
+        value["app_id"] = service["appId"]
+        synonyms.append(service["appId"])
+    if "alt_title" in service:
+        synonyms += service["alt_title"]
+    return FuzzySlotMatchOption(value, synonyms)
```

Comment:
Lets return the data and the processed synonyms

```suggestion
    return {
      "data": value,
      "synonyms": synonyms
    }
```